packages/block-editor/package.json

```diff
@@ -31,6 +31,7 @@
     "@wordpress/icons": "^9.3.0",
     "classnames": "^2.3.1",
     "lodash": "^4.17.21",
+	"react-phone-number-input": "^3.2.11",
```

Comment:
Now we created a wrapper component inside the `blocks` package we don't need this dependency here.

Reply:
When I removed it, the linter wouldn't let me commit. It said it was needed here.